src/js/live-tracker.js

```diff
@@ -1,19 +1,43 @@
 import Component from './component.js';
-import median from './utils/median.js';
```

Comment:
I think if we remove it here, we can't just remove `media.js` as it isn't used anywhere else, I don't think?